src/ecalc/libraries/libecalc/common/libecalc/common/utils/rates.py

```diff
@@ -214,7 +214,9 @@ def __setitem__(self, indices: Union[slice, int, List[int]], values: Union[TimeS
                     f"Could not update timeseries, Combination of indices of type '{type(indices)}' and values of type '{type(values)}' is not supported"
                 )
 
-    def __eq__(self, other) -> bool:
+    def __eq__(self, other: object) -> bool:
```

Comment:
`object` or `Any`? In general we have used any, what is the reason to use object here?

Reply:
I believe this is due to "liskov substitution"?  Since the class inherits from object, it should also support object in all functions. Mypy does not like 'any', but not sure it is any better. 

Googled around and found some support for using 'object' but I am open for discussion

https://stackoverflow.com/questions/37557411/why-does-defining-the-argument-types-for-eq-throw-a-mypy-type-error

Reply:
https://mypy.readthedocs.io/en/latest/dynamic_typing.html#any-vs-object

Mypy reccomends 'object' over 'any'. Will keep as-is for now. May revert later